pkg/network/dns/ebpf.go

```diff
@@ -32,6 +41,11 @@ func newEBPFProgram(c *config.Config) (*ebpfProgram, error) {
 	}
 
 	mgr := &manager.Manager{
+		Maps: []*manager.Map{
+			{Name: string(probes.ConnMap), MapOptions: manager.MapOptions{PinPath: filepath.Join(probes.TracerPinPath, string(probes.ConnMap))}},
```

Comment:
FYI you don't need to pin maps if you just need to share them within the same process. Here's an example: https://github.com/DataDog/datadog-agent/blob/2f6c9821ab590afdeaafd45bcc1b9dae00f84013/pkg/network/http/ebpf_ssl.go#L137

Reply:
I actually think we don't want to pin maps, because that makes them live beyond the process.

Reply:
pin should not persist as we manager.Close(cleanAll) and I cleaned up the /sys/bpf temp dir on close
But I agreed, I shared map/fd between the tracer and the classifier